### ISSUE TITLE
[Accessibility][Screen Reader][JAWS] - Activating the focused links not working with JAWS 

### ISSUE THREAD

**Issue by UncleZen** [microsoft/BotFramework-WebChat#3176]
OS: Windows 10
WebChat: v. 4.9.0 CDN version
Screen reader: JAWS, Narrator
Browser: MS  Edge Version 83.0.478.37, IE 11, Chrome Version 81.0.4044.138 

## Describe the bug

When the screen readers in general read the content of a bubble, they do not announce the element as being a link and allow the user to press enter and navigate to Url. Activating links is possible only after tabbing back and forth, but only with Narrator. JAWS fails most of the time to open the link.  

I was able to observe that if a link is contained within an adaptive card, once focused JAWS will activate it all the time.

**Comment by compulim** [comment]
Thanks for reporting. I think we need to look at all `aria-hidden="true"` and make sure they are not overused. In this case, the Markdown content is being hidden from screen reader, which we should not.

![image](https://user-images.githubusercontent.com/1622400/83293548-c595f580-a1a0-11ea-9cda-e4e8beba664c.png)

**Comment by UncleZen** [comment]
@compulim 
Thanks for the prompt reply. Any estimates as to when this issue would be fixed?

**Comment by compulim** [comment]
This is fixed in #3287.

**Comment by UncleZen** [comment]
> This is fixed in #3287.

@compulim 
When will this fix be published?

**Comment by compulim** [comment]
It is expected to release as a patch release, 4.9.2 and planned for next week as we are doing localization. 😊
